packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -866,7 +872,8 @@ class _BodyBuilder extends StatelessWidget {
           : metrics.padding.bottom;
 
         final double top = extendBodyBehindAppBar
-          ? math.max(metrics.padding.top, bodyConstraints.appBarHeight)
+          ? math.max(metrics.padding.top,
+            bodyConstraints.appBarHeight + bodyConstraints.materialBannerHeight)
```

Comment:
Fix the indentation to something like:

```suggestion
          ? math.max(metrics.padding.top,
              bodyConstraints.appBarHeight + bodyConstraints.materialBannerHeight)
```